user-manual.md

```diff
@@ -300,338 +243,161 @@ Chatbot: Nimmst du Drogen?
+#### Beispiel 3 "Anfrage nach einer Unterkunft mit direkter Eingabe eines Teils der benötigten Informationen (männlich, volljährig)"
 
 ```
-Eingabe: Ich suche eine Unterkunft.
-Chatbot: Ich benötige noch einige allgemeine Informationen von dir. Diese werden nach dieser Unterhaltung nicht gespeichert.
-Chatbot: Wie alt bist du?
-Eingabe: 32.
-Chatbot: Welchem Geschlecht fühlst du dich zugehörig?
-Eingabe: Männer.
-Chatbot: Hast du ein Haustier?
-Eingabe: Nein.
+Eingabe: Ich bin männlich, 32 und suche einen Platz zum schlafen
+Chatbot: Ok, zum Thema 'Unterkunft' benötige ich noch einige allgemeine Informationen von dir. Diese werden nach dieser Unterhaltung nicht gespeichert.
+Hast du ein Haustier?
+Eingabe: Nein
 Chatbot: Nimmst du Drogen?
-Eingabe: Ja.
+Eingabe: Nein
 Chatbot: Folgende Informationen wurden erfasst (werden nicht abgespeichert):
         - Alter: 32
-        - Geschlecht: nännlich
-        - Haustier: nein
-        - Drogen: ja
-        - Volljährig: ja
-        - Unter 22 (jugendlich): nein
-Chatbot: Brauchst du eine Notunterkunft?
-Eingabe: ja.
-Chatbot: Hier kannst du eine Notunterkunft (Deep Link) für Menschen mit Drogenproblemen finden.
+        - Geschlecht: weiblich
+        - Haustier: False
+        - Drogen: False
+        - Volljährig: True
+        - Unter 22 (jugendlich): False
+        Hier findest du die Notunterkünfte für Frauen.
+        ...
+        (Deep Link)
 ```
 
-##### Zusammenfassung Beispiel 8:
-
-In diesem Beispiel sucht ein erwachsener Mann eine Notunterkunft. Er nimmt Drogen und hat kein Haustier. Der Endpunkt ist eine Auflistung von Notunterkünften die Menschen ohne Haustier aber mit Drogenproblemen aufnehmen. Hat ein Hilfesuchender ein Haustier und ein Drogenproblem wird bei der Auswahl der Notunterkünfte das Haustier priorisiert.
-
-#### Beispiel 9 "Notunterkunft ohne Drogen und ohne Haustier (Shortcut)"
-
-```
-Eingabe: Ich bin 37, ein Mann, habe keinen Hund, nehme keine Drogen und brauche einen Platz zum Schlafen
-Chatbot: Folgende Informationen wurden erfasst (werden nicht abgespeichert):
-        - Alter: 37
-        - Geschlecht: nännlich
-        - Haustier: nein
-        - Drogen: nein
-        - Volljährig: ja
-        - Unter 22 (jugendlich): nein
-Chatbot: Hier kannst du Notunterkünfte (Deep Link) für Männer finden.
-```
+## Medizin
 
-##### Zusammenfassung Beispiel 9:
+Im Anwendungsfall `Medizin` kann man sich die Adressen und Kontaktdaten zu medizinischen Hilfestellen entsprechend dem Versicherungsstatuses der/des Anwender:in anzeigen lassen.
+Im Falle eines Notfalles, können im auch Notruf-Informationen abgefragt werden.
 
-In diesem Beispiel sucht ein erwachsener Mann eine Notunterkunft. 
-Er nimmt keine Drogen und hat kein Haustier. 
-Diese Informationen sind alle in seiner ersten Eingabe enthalten und werden vom Bot entsprechend extrahiert. So ist ein direkter Sprung (Shortcut) zum Endpunkt möglich. 
+### Mögliche Eingaben um allgemein das Thema Medizin aufzurufen
 
```

Comment:
Warum wurden die Beispiele vier bis acht gelöscht?

Reply:
Weil sie meiner Ansicht nach ähnlich wie bei unserer Demo im Vortrag keinen Mehrwert geboten haben bzgl. der Eingabe. Es war immer der komplett gleiche Chatverlauf. Das fand ich zu redundant und habe sie deswegen raus genommen.

Reply:
Visualisierung aller Pfade
rasa visualize noch hinzufügen um alle Endpunkte anzuzeigen.

"Alle Endpunkte können in visualize nachgesehen werden"